src/core/js/models/adaptModel.js

```diff
@@ -536,9 +542,9 @@ define([
         }
         siblings = _.reject(Adapt[this._siblings].where({
           _parentId: this.get('_parentId')
-        }), function (model) {
+        }), model=>{
```

Comment:
This could be done in one fell swoop with a native `.filter()`?
```js
siblings = Adapt[this._siblings].filter(model => {
  return model.get('_parentId') === this.get('_parentId') &&
    model.get('_id') !== this.get('_id');
});
```